serde/src/java/org/apache/hadoop/hive/serde2/objectinspector/primitive/PrimitiveObjectInspectorUtils.java

```diff
@@ -737,6 +737,9 @@ public static int getInt(Object o, PrimitiveObjectInspector oi) {
       result = (int) (((TimestampObjectInspector) oi)
           .getPrimitiveWritableObject(o).getSeconds());
       break;
+    case TIMESTAMPLOCALTZ:
+      result = (int) (((TimestampLocalTZObjectInspector) oi).getPrimitiveWritableObject(o).getSeconds());
```

Comment:
I don't understand what this change has to do with this patch
I guess the new udf uses this category; what category it used the old udf? 
if it was a bug we could address that separately